services/director-v2/src/simcore_service_director_v2/cli/_client.py

```diff
@@ -30,7 +23,7 @@ async def toggle_service_observation(
     ) -> Response:
         return await self.client.patch(
             self._get_url(f"/services/{node_uuid}/observation"),
-            json=dict(is_disabled=is_disabled),
+            json={"is_disabled": is_disabled},
```

Comment:
since Mattermost is broken.

This is what I do not really understand with this whole thin vs fat client story:
- everytime you add an entrypoint you need to edit both clients everytime which in terms of maintainability is bad
- if the thin client is pydantic-less kind of, should it not only have the base calls such as post/delete/patch/put/get with the retry/expect_status stuff?

anyway. that is my thought.

Reply:
So:
- yes, your first statement is correct and I agree with it. I think it's a tradeoff between encapsulation and maintainability.
- As you said this thin client only guarantees that the returned response has the expected status code and it also applies a retry before giving up. You can see that all the interfaces return a `Response` instance.